scylla-cdc/src/checkpoints.rs

```diff
@@ -187,7 +187,7 @@ impl CDCCheckpointSaver for TableBackedCheckpointSaver {
     async fn load_last_generation(&self) -> anyhow::Result<Option<GenerationTimestamp>> {
         let generation = self
             .session
-            .query(
+            .query_unpaged(
                 format!(
                     "SELECT generation FROM {}
                     WHERE stream_id = ?",
```

Comment:
:thinking: This is a SELECT. I'm unsure if it's acceptable that we use `query_unpaged` for this. Is the query always expected to return at most one row? What about the tombstones encountered? @Lorak-mmk

Reply:
I'm don't know how this works with cdc tables (are there tombstones there?). @piodul

Reply:
Why is it a concern here? This PR only bumps up the driver version, let's not use it to fix bugs if there are any. `query` was renamed to `query_unpaged`, so let's do only that here.

Besides, look at the schema of the checkpoint table. It only has a partition key (`stream_id`) and no clustering key. This query retrieves a single partition, which will be a single row.

Reply:
This convinces me, changing query types can be done at another time (if at all).

Reply:
> Why is it a concern here? This PR only bumps up the driver version, let's not use it to fix bugs if there are any.

I slightly disagree. The point of the changes introduced in 0.14 (`query` -> `query_unpaged`/`query_single_page`) was to require users to make conscious decision wrt paging, preferably straight upon migration. In this library we are users of the driver and thus we should reconsider our use of paging methods right upon migration.